lib/tasks/webpacker/compile.rake

```diff
@@ -26,7 +26,18 @@ end
 namespace :webpacker do
   desc "Compile JavaScript packs using webpack for production with digests"
   task compile: ["webpacker:verify_install", :environment] do
-    Webpacker.with_node_env("production") do
+    node_env = "production"
```

Comment:
```
node_env = ENV["NODE_ENV"] || "production"

if node_env != "production"
  $stdout.puts "Using NODE_ENV=#{ENV["NODE_ENV"]} from local "\
     "environment instead of default NODE_ENV=production"
end
```